test/setup.js

```diff
@@ -91,21 +90,81 @@ function screenshotCreateOrCompare (app, t, name) {
   })
 }
 
+// Compares two PNGs, ignoring any transparent regions in bufExpected.
+// Returns true if they match.
+function compareIgnoringTransparency (bufActual, bufExpected) {
+  // Common case: exact byte-for-byte match
+  if (Buffer.compare(bufActual, bufExpected) === 0) return true
+
+  // Otherwise, compare pixel by pixel
+  const pngA = PNG.sync.read(bufActual)
+  const pngE = PNG.sync.read(bufExpected)
+  if (pngA.width !== pngE.width || pngA.height !== pngE.height) return false
+  const w = pngA.width
+  const h = pngE.height
+  const da = pngA.data
+  const de = pngE.data
+  for (let y = 0; y < h; y++) {
+    for (let x = 0; x < w; x++) {
+      const i = (y * w + x) * 4
+      if (de[i + 3] === 0) continue // Skip transparent pixels
+      if (da[i] !== de[i] || da[i + 1] !== de[i + 1] || da[i + 2] !== de[i + 2]) return false
+    }
+  }
+  return true
+}
+
 // Resets the test directory, containing config.json, torrents, downloads, etc
-function wipeTestDataDir () {
-  fs.removeSync(TEST_DATA_DIR)
-  fs.mkdirpSync(TEST_DOWNLOAD_DIR) // Downloads/ is inside of TEST_DATA_DIR
+function resetTestDataDir () {
+  fs.removeSync(config.TEST_DIR)
+  // Create TEST_DIR as well as /Downloads and /Desktop
+  fs.mkdirpSync(config.TEST_DIR_DOWNLOAD)
+  fs.mkdirpSync(config.TEST_DIR_DESKTOP)
+}
```

Comment:
We can put it in `/tmp` instead of here but we should still clean it up ourselves. Otherwise, a single run of the integration tests will produce 10+ different app folders, each with their own `config.json`, Chrome cache, torrent and poster files, etc

I think it's fine